src/BundledBindings/HttpTriggerBinding.cs

```diff
@@ -25,30 +25,21 @@ public HttpTriggerBinding()
             {
                 Name = parameter.Name.VariablePath.UserPath
             };
-            //Todo: Support for named arguments: https://github.com/Azure/azure-functions-powershell-library/issues/11
```

Comment:
Could you please include a comment in the parser code on why we decided to use name arguments instead of positional arguments? I personally like this approach better. Thank you.

Reply:
I did already add a comment at WorkerIndexingHelper.cs line 236, but I'll go a little more in depth

Reply:
Thank you for adding more details.